src/Compilers/CSharp/Portable/Symbols/Source/SourceFieldSymbol.cs

```diff
@@ -202,6 +200,7 @@ private CustomAttributesBag<CSharpAttributeData> GetAttributesBag()
                 return bag;
             }
 
+            var declarationDiagnostics = DeclaringCompilation.DeclarationDiagnostics;
```

Comment:
>var declarationDiagnostics = DeclaringCompilation.DeclarationDiagnostics; [](start = 12, length = 73)

It doesn't look like this change is needed and there are no other changes in this file that are related to the purpose of this PR. Please revert/remove from the PR.